static/styles/main.scss

```diff
@@ -21,6 +21,7 @@
 @import 'partials/_images.scss';
 @import 'partials/_account.scss';
 @import 'partials/_success.scss';
+@import 'partials/hourglass';
```

Comment:
This should be `partials/_hourglass.scss` right?

Reply:
Ohh im sorry. Yes that was meant to be '@import `partials/hourglass'.scss;`